<launch>
  <!--<node name="rviz" pkg="rviz" type="rviz" args="-d $(find MSTC_Star_update)/rviz/navigate.rviz" />-->

  <param name = "/allocate_method" value = "DARP" />
  <param name = "/MST_shape" value = "DINIC" />
  <param name = "/useROSPlanner" value = "false" />
  <param name = "/coverAndReturn" value = "false" />
  <param name = "/GA_max_iter" value = "200000" />
  <param name = "/unchanged_iter" value = "2000" />
  <param name = "/hp_max_iter" value = "5000" />
  <param name = "/free_threshold" value = "0.0" /> <!-- [0; 100] 0 is free, 100 is occupied, at what point is a cell considered occpuied for cleaning after interpolation-->


  <param name = "/hp_max_iter" value = "5000" />

  <node name="coverage_path_server" pkg="TMSTC_Star" type="coverage_path_server" output="screen"/>
</launch>